Fixed the engineinfo key format in the script eschema. (#3622)
demisto/demisto-sdk: CHANGELOG.md

* Fixed an issue where the **validate -g** failed reading a `.pack-ignore` file that contained only newlines and spaces.
* Demisto-SDK will now exit gracefully with an appropriate error message when *git* is not installed.
* Fixed the support in **validate** for `svg` images to have their theme suffix.

> * Fixed the **engineinfo** key type in the script schema. This field specifies on which engine the script will run.

Reviewer: ```suggestion
* Updated the **engineinfo** type in the script schema.
```
